x-pack/plugins/security_solution/cypress/screens/search_bar.ts

```diff
@@ -32,3 +32,10 @@ export const GLOBAL_SEARCH_BAR_FILTER_ITEM = '#popoverFor_filter0';
 export const GLOBAL_SEARCH_BAR_FILTER_ITEM_AT = (value: number) => `#popoverFor_filter${value}`;
 
 export const GLOBAL_SEARCH_BAR_PINNED_FILTER = '.globalFilterItem-isPinned';
+
+export const GLOBAL_KQL_INPUT =
+  '[data-test-subj="filters-global-container"] [data-test-subj="unifiedQueryInput"] textarea';
+
+export const AUTO_SUGGEST_AGENT_NAME = `[data-test-subj="autocompleteSuggestion-field-agent.name-"]`;
```

Comment:
NIT: If we want to make sure that there is a field suggesting a value and we don't care about the value itself. I would rename the name of the selector and use the following one: '[data-test-subj^=local-autocompleteSuggestion-field-]'. In this way you are telling Cypress to find for a locator starting by that dataTestSubj without taking into account if there is about any specific field, making the test more reliable in case of any change on the first value displayed in the future,

Reply:
but I want to make sure its getting the correct field values, i am going to leave it if thats ok

Reply:
Sure!! :)